lib/codegen-meta/src/base/mod.rs

```diff
@@ -0,0 +1,3 @@
+//! Definitions for the base Cretonne language.
```

Comment:
Just noticed a few instances of the name "Cretonne" snuck through, I can rectify these.